e2e/databases.spec.ts

```diff
@@ -14,12 +14,14 @@ test.describe('databases', () => {
     })
 
     test('copy/rename database', async ({ page }) => {
-        test.skip()
-
         await page.getByRole('link', { name: 'Databases' }).click()
+        const firstRowButton = page
+            .locator('table[data-test="dhis2-uicore-datatable"] tbody tr:first-child td[data-test="dhis2-uicore-datatablecell"]:last-child button[data-test="dhis2-uicore-button"]')
```

Comment:
tr button[data-test="dhis2-uicore-button"]

The above selector should do the same